packages/opentelemetry-semantic-conventions/src/resource/index.ts

```diff
@@ -13,4 +13,144 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-export * from './SemanticResourceAttributes';
+export {
```

Comment:
@dyladan Should I exclude this package from the export cleanup because #4690? 😅

Reply:
I updated this branch to _not_ change the semantic conventions export-all yet. Maybe that can happen as a part of #4690 and the code-generation?

Reply:
There are two levels of re-export, at least in the current semconv package:

* ♻️ [src/index.ts](https://github.com/open-telemetry/opentelemetry-js/blob/v1.25.1/packages/opentelemetry-semantic-conventions/src/index.ts#L17-L18)
    * ♻️ [src/resource/index.ts](https://github.com/open-telemetry/opentelemetry-js/blob/v1.25.1/packages/opentelemetry-semantic-conventions/src/resource/index.ts#L16)
        * ⬆️ src/resource/SemanticResourceAttributes.ts (the real consts)
    * ♻️ [src/trace/index.ts](https://github.com/open-telemetry/opentelemetry-js/blob/v1.25.1/packages/opentelemetry-semantic-conventions/src/trace/index.ts#L16)
        * ⬆️ src/resource/SemanticAttributes.ts (the real consts)

I see that [the proposed new structure for semconv will have only one-level](https://github.com/open-telemetry/opentelemetry-js/pull/4690/files#diff-6fd5311c290390febeec38723efb1b45ae12c26b458a00d16fbdc6dd669d401eR22-R23), so maybe it's fine to keep the two levels here in the deprecated paths?